### DISCUSSION TITLE
Why new branches doesn't reflect schema of main

### DISCUSSION THREAD

**Discussion by HCoban** [planetscale/discussion#25] [Q&A] [answered]
Hello,

My main branch has two tables, `reminders` and `users`. When I create a new branch "myfeature" off of main, this branch only includes `reminders` table. 

I would expect `users` table to be in the "myfeauture" branch as well since it was based off of main when main was already having `users` table.

I reproduced this issue by creating multiple branches and all have the same issue. Is this the expected behavior or there was a step I missed?

**Comment by Anders-PlanetScale** [comment]
Halil,

This appears to be a bug related to using the demo-dataset I have submitted a bug report to our engineering team to investigate. If you create a new DB without the demo data you should be able to see schema changes from previous deployed branches in new branches you create off of main.

**Comment by nickvanw** [comment, marked as answer]
Hey @HCoban!

Sorry about this - you're definitely not doing anything wrong, this behavior isn't expected!

Unfortunately, our "demo database" code was a little overzealous, which is why it keeps overriding your new branch to have _only_ the `reminders` table 😅. 

Fortunately, I believe I've isolated the problem and we're working on a fix right now. As @Anders-PlanetScale said, this bug doesn't appear when using databases that have been created without the demo dataset.

Sorry for the trouble!

**Reply by HCoban** [reply]
Hello @nickvanw 

Thank you for looking into this. Yes, you are right, this bug doesn't appear when using databases that have been created without the demo dataset. 

I created a new DB and I don't have this issue anymore.

**Reply by nickvanw** [reply]
We just released a bug fix to the original problem as well, which should hopefully stop the next person from experiencing this as well. Thank you again for the report!
